### DISCUSSION TITLE
DateTime columns become Time when selected

### DISCUSSION THREAD

**Discussion by 3ynm** [jeremyevans/sequel#2046] [General]
I'm creating DateTime columns on my migrations, but when I fetch an object those columns have the Time class. How could I have DateTime attributes returned instead? I'm using models.

**Comment by jeremyevans** [comment]
`Sequel.datetime_class = DateTime` will make Sequel return timestamp values as DateTime instead of Time, assuming you are using an adapter that supports that.
